server/metadata/database.go

```diff
@@ -101,3 +95,148 @@ func (b *DatabaseName) IsMainBranch() bool {
 	}
 	return false
 }
+
+// DatabaseSubspace is used to store metadata about Tigris databases.
+type DatabaseSubspace struct {
+	metadataSubspace
+}
+
+const dbMetaValueVersion int32 = 1
+
+func newDatabaseStore(nameRegistry *NameRegistry) *DatabaseSubspace {
+	return &DatabaseSubspace{
+		metadataSubspace{
+			SubspaceName: nameRegistry.EncodingSubspaceName(),
+			KeyVersion:   []byte{encKeyVersion},
```

Comment:
Similar to collection, have a key version for this subspace as well.

Reply:
Okay, in your previous diff I think collection had its own keyVersion so I got confused.